pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -150,10 +152,25 @@ func ProvideService(
 		if err != nil {
 			return nil, fmt.Errorf("creating http client for GCOM: %w", err)
 		}
-		s.gcomService = gcom.New(gcom.Config{ApiURL: cfg.GrafanaComAPIURL, Token: cfg.CloudMigration.GcomAPIToken}, httpClientGcom)
+		gcomS := gcom.New(gcom.Config{ApiURL: cfg.GrafanaComAPIURL, Token: cfg.CloudMigration.GcomAPIToken}, httpClientGcom)
```

Comment:
Could these two lines be moved directly into the `else` block? so you could do:
```go
s.authApiService = gcom.New(...)
```

Reply:
yeah you are right. I wanted to reuse the gcom service, but it is cleaner if we create a new one, and should change the final behave. And when we clean this code we will just need to delete the else block
fixed in https://github.com/grafana/grafana/pull/96581/commits/cee4c056441a8fbd485e7a2587accb94de3ed466